index.js

```diff
@@ -35,42 +50,45 @@ function setDate() {
   };
 }
 
-addCurrencyBtn.addEventListener("click", function addCurrency(event) {
+addCurrencyBtn.addEventListener("click", function (event) {
   event.preventDefault();
```

Comment:
You can remove the `event` part here if you would add `type="button"` in your HTML. Without explicit `type` the form assumes the button is of `type="submit"` and acts this way upon click.
```suggestion
addCurrencyBtn.addEventListener("click", function () {
```